File: GTTS/INL.MessagingService/Models/NotificationRecipient_Item.cs
using System;

namespace INL.MessagingService.Models
{
    public class NotificationRecipient_Item : INotificationRecipient_Item
    {
        public long NotificationID { get; set; }
        public int AppUserID { get; set; }
        public DateTime? ViewedDate { get; set; }
        public DateTime? EmailSentDate { get; set; }
        public DateTime ModifiedDate { get; set; }
        public bool Unread { get; set; }
        public string AppUser { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/notification-recipient_item.ts


import { INotificationRecipient_Item } from './inotification-recipient_item';

export class NotificationRecipient_Item implements INotificationRecipient_Item {
  
	public NotificationID: number = 0;
	public AppUserID: number = 0;
	public ViewedDate?: Date;
	public EmailSentDate?: Date;
	public ModifiedDate: Date = new Date(0);
	public Unread: boolean = false;
	public AppUser: string = "";
  
}


